golang/Release/metadata.yml

```diff
@@ -0,0 +1,6 @@
+suggested: false
```

Comment:
whats the purpose of these flags?

Reply:
This is used to show the suggested box around the pipeline in the UI